Platform/CMakeLists.txt

```diff
@@ -14,10 +14,11 @@ if(WIN32)
     list(APPEND PLATFORM_HEADERS Win/Win.h)
 else()
     list(APPEND PLATFORM_SOURCES Lin/Lin.cpp)
-    include_directories(${SDL_INCLUDE_DIRS})
 endif()
 
+include_directories(${SDL_INCLUDE_DIR})
```

Comment:
could you move that to sdl2.cmake  so this is available project-wide?

Reply:
done